students/randi_peterson/session06/test_mailroom.py

```diff
@@ -0,0 +1,89 @@
+from mailroom import *
+from os import listdir
+
+#-----TEST Sending Thank You-----
+#Test 'list' -- skipped because involves printing
+
+#Test 'exit'
+def test_exit(test_dict):
+    assert give_thanks(test_dict,'exit') == None
+
+#Test existing name
+def test_existing_name(test_dict):
```

Comment:
it's best to have your test name include what function you're testing -> `test_add_to_list_existing_name`